build.sh.tpl

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```

Comment:
Isn't bash guaranteed to be in /bin/? I mean we usually write `#!/bin/bash` is there any advantage of writing it the way you did?

Reply:
It's an old habit from FreeBSD where bash was at /usr/local/bin/bash and /bin/csh used as the default shell